src/packages/cli/package.json

```diff
@@ -21,12 +22,11 @@
 	"dependencies": {
 		"@exogee/graphweaver-admin-ui": "workspace:*",
 		"@exogee/graphweaver-builder": "workspace:*",
-		"@octokit/core": "4.2.1",
-		"@octokit/plugin-throttling": "6.0.0",
-		"@octokit/rest": "19.0.11",
+		"@exogee/graphweaver-mikroorm": "workspace:*",
 		"chokidar": "3.5.3",
 		"esbuild": "0.15.5",
 		"inquirer": "9.2.1",
+		"ora": "5.4.1",
```

Comment:
Nice! A terminal spinner.

Reply:
Postgres DBs can take quite a bit of time im not sure why. SQLite and MySql are super quick